enterprise/frontend/src/embedding/auth-common/validate-session-token.ts

```diff
@@ -0,0 +1,35 @@
+import * as MetabaseError from "embedding-sdk/errors";
+
+export function validateSessionToken(session: any) {
```

Comment:
This is exactly the same as before, just moved into a function so it can be used from multiple places.